services/core/MasterDriverAgent/master_driver/interfaces/bacnet.py

```diff
@@ -215,16 +236,24 @@ def parse_config(self, configDict):
 
         for regDef in configDict:
             #Skip lines that have no address yet.
-#             if not regDef['Point Name']:
-#                 continue
+            if not regDef['Volttron Point Name']:
+                continue
 
             io_type = regDef['BACnet Object Type']
             read_only = regDef['Writable'].lower() != 'true'
             point_name = regDef['Volttron Point Name']
+
+            # checks if the point is flagged for change of value
+            is_cov = False
+            if 'COV Flag' in regDef:
+                if regDef['COV Flag'] == "True":
```

Comment:
Convert everything to lower case so things like "TRUE" will work as expected.
This can also be rewritten as a single statement:
is_cov = regDef.get("COV FLag", 'false').tolower() == "true"